pkg/releasesync/releasesync.go

```diff
@@ -76,6 +90,27 @@ func (r *releaseSyncOptions) Run() error {
 	return nil
 }
 
+func (r *releaseSyncOptions) buildReleaseTag(architecture, release string, tag ReleaseTag) *models.ReleaseTag {
+	releaseDetails := r.fetchReleaseDetails(architecture, release, tag)
+	releaseTag := releaseDetailsToDB(architecture, tag, releaseDetails)
+
+	// We skip releases that aren't fully baked (i.e. all jobs run and changelog calculated)
+	if releaseTag == nil || (releaseTag.Phase != api.PayloadAccepted && releaseTag.Phase != api.PayloadRejected) {
```

Comment:
I think there's a third state, PayloadFailed, which I think happens when something goes wrong in the ART pipeline or similar. Do we need to handle that here? (I see that this is just code being moved so maybe not a concern for this PR)

Reply:
Only if we want those to show up in Sippy.  Right now they would be skipped which seems reasonable unless we want historical context for pipeline failures

Reply:
Gotcha, thanks!